format_checker/common_checks.py

```diff
@@ -92,7 +92,7 @@ def check_sort(filename, log):
     diff = subprocess.check_output(command, shell=True).decode("utf-8")
     if diff != "":
         log_esp_error(filename, log, "The file is not properly ordered")
-        print("Refer to IDoFT readme for how pr-data.csv should be sorted: https://github.com/TestingResearchIllinois/idoft#to-contribute-a-newly-detected-flaky-test")
+        print("Refer to IDoFT readme for how pr-data.csv, gr-data.csv should be sorted: https://github.com/TestingResearchIllinois/idoft#to-contribute-a-newly-detected-flaky-test")
```

Comment:
Could this print only the file name being checked (and where the check is failing) rather than printing both file names?

Reply:
fixed